### PR TITLE
Add contract first development section in rest-openapi doc

### PR DESCRIPTION
**Pull request by zhfeng**

### CODE REVIEW

File: extensions/rest-openapi/runtime/src/main/doc/usage.adoc [hunk 1]
```diff
@@ -24,4 +24,55 @@ When using the `classpath` resource locator with native code, the path to the Op
 [source]
 ----
 quarkus.native.resources.includes=openapi.json
-----
\ No newline at end of file
+----
+
+=== Contract First Development
+The model classes generation has been integrated with `quarkus-maven-plugin`. So it's no need to use `swagger-codegen-maven-plugin`, instead to put your contract files in the `src/main/openapi` with `.json` suffix. And add `generate-code` goal in `quarkus-maven-plugin` like:
```
Comment (jamesnetherton) [inline extensions/rest-openapi/runtime/src/main/doc/usage.adoc:30]:
```suggestion
The model classes generation has been integrated with the `quarkus-maven-plugin`. So there's no need to use the `swagger-codegen-maven-plugin`, instead put your contract files in `src/main/openapi` with a `.json` suffix. And add the `generate-code` goal to the `quarkus-maven-plugin` like:
```

File: extensions/rest-openapi/runtime/src/main/doc/usage.adoc [hunk 2]
```diff
@@ -24,4 +24,55 @@ When using the `classpath` resource locator with native code, the path to the Op
...
+        <execution>
+            <goals>
+                <goal>generate-code</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+----
+
+It requires to specific the package name of the model classes by using `quarkus.camel.openapi.codegen.model-package` property of the `application.properties` file. For example:
```
Comment (jamesnetherton) [inline extensions/rest-openapi/runtime/src/main/doc/usage.adoc:47]:
```suggestion
It requires a specific package name for the model classes by using the `quarkus.camel.openapi.codegen.model-package` property of the `application.properties` file. For example:
```

**Review by jamesnetherton — APPROVED** [review]
Thanks a lot @zhfeng. I left a couple of suggestions.

File: extensions/rest-openapi/runtime/src/main/doc/usage.adoc (hunk 1 above)
Comment (zhfeng) [reply extensions/rest-openapi/runtime/src/main/doc/usage.adoc]: Done

File: extensions/rest-openapi/runtime/src/main/doc/usage.adoc (hunk 2 above)
Comment (zhfeng) [reply extensions/rest-openapi/runtime/src/main/doc/usage.adoc]: Done.

**Pull request merged**